DESCRIPTION

```diff
@@ -42,6 +42,9 @@ Suggests:
     callr,
     caret,
     covr,
+    curl,
+    dplyr,
```

Comment:
This PR adds A LOT of packages here. I put all the packages for monitoring in `Suggests` because I am up against 20 packages in Imports already and I can sort of make the argument that people may want to deploy a model but not monitor it. I am very open to ideas for reducing this somehow.

Reply:
I'm only using dplyr in examples and tests FWIW 🤷‍♀️